### PR TITLE
docs: adding description in README.md

### PR DESCRIPTION
**Pull request by CTAPCKPIM**
- Adding description

### CODE REVIEW

**Comment by zolotokrylin** [comment]
@almon-s please review the copy

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+The main commands that are used when interacting in a project.  
+    For __compile__ the contract:  
+```plaintext
+    npm run compile  
+```  
+For __testing__ the contract:
+```plaintext
+    npm run test  
+```  
+Shows smart the contract __coverage__;
```
Comment (almon-s) [inline README.md:18]: Shows the smart contract*

**Review by almon-s — COMMENTED** [review]
Please fix typo

**Comment by CTAPCKPIM** [comment]
@almon-s 
Done

**Review by markholdex — APPROVED** [review]

**Pull request merged**